frontend/src/app/main/ui/workspace/sidebar/options/component.cljs

```diff
@@ -54,8 +55,23 @@
         do-update-component #(do
                                (st/emit! (dwc/start-undo-transaction))
                                (st/emit! (dwl/update-component id))
-                               (st/emit! (dwl/sync-file current-file-id))
+                               (st/emit! (dwl/sync-file current-file-id current-file-id))
                                (st/emit! (dwc/commit-undo-transaction)))
+        confirm-update-remote-component #(do
+                                           (st/emit! (dwl/update-component id))
+                                           (st/emit! (dwl/sync-file current-file-id
+                                                                    (:component-file values)))
+                                           (st/emit! (dwl/sync-file (:component-file values)
+                                                                    (:component-file values))))
```

Comment:
the same as before, can be defined using emitf

Reply:
done, same as above